Auto update failing for vulkan-validation-layers

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [vulkan-validation-layers](https://github.com/termux/termux-packages/blob/c4b1f823a73c79a62368e1edbd85db6e2ec474bf/packages/vulkan-validation-layers) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating vulkan-validation-layers [Current version: 1.3.290]
INFO: package being updated to 1.3.293.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 3312k    0 3312k    0     0  4858k      0 --:--:-- --:--:-- --:--:-- 4858k
100 5183k    0 5183k    0     0  5292k      0 --:--:-- --:--:-- --:--:-- 6302k

You are about to commit these changes:

--------------------
diff --git a/packages/vulkan-validation-layers/build.sh b/packages/vulkan-validation-layers/build.sh
index 81e2ac2c9..dec903536 100644
--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.290"
+TERMUX_PKG_VERSION="1.3.293"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=37fd611f1c5c5879a68c6f63b09e600a3858f519aa147a2535961fa3d8e889ae
+TERMUX_PKG_SHA256=5cf50380e1f65c19f74cfcfb577906910d1c8c51405d37fe0c736351f73c1b07
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"
--------------------

bump(main/vulkan-validation-layers): 1.3.293


Not committing to Git!

INFO: Trying to build package.
INFO: Package vulkan-validation-layers exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building vulkan-validation-layers for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  80279      0 --:--:-- --:--:-- --:--:-- 80576
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   4441      0 --:--:-- --:--:-- --:--:--  4430
100   833  100   833    0     0   4438      0 --:--:-- --:--:-- --:--:--  4430
gpg: Signature made Wed 21 Aug 2024 12:35:32 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  76523      0 --:--:-- --:--:-- --:--:-- 76994
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   4989      0 --:--:-- --:--:-- --:--:--  5018
gpg: Signature made Sun 18 Aug 2024 10:35:05 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  84868      0 --:--:-- --:--:-- --:--:-- 85194
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   4676      0 --:--:-- --:--:-- --:--:--  4679
gpg: Signature made Tue 20 Aug 2024 10:07:22 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@27 if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.2 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.6-1 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  543k  100  543k    0     0  6694k      0 --:--:-- --:--:-- --:--:-- 6708k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8988  100  8988    0     0   119k      0 --:--:-- --:--:-- --:--:--  120k
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.2 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.2 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4.20231001-5 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4.20231001-5 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4.20231001-5 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4.20231001-5 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.24 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.07.02 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.07.02 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.46.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.46.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.46.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.43 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.3 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.1 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2024.1 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.17.0 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.17.0 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.17.0 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.10 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.11.9-5 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.11.9-5 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.11.9-5 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency libxml2@2.13.3 if necessary...
Found libxml2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2/libxml2_2.13.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  455k  100  455k    0     0  3919k      0 --:--:-- --:--:-- --:--:-- 3930k
extracting libxml2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-utils@2.13.3 if necessary...
Found libxml2-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-utils/libxml2-utils_2.13.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 30952  100 30952    0     0   330k      0 --:--:-- --:--:-- --:--:--  332k
extracting libxml2-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python@2.13.3 if necessary...
Found libxml2-python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python/libxml2-python_2.13.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  189k  100  189k    0     0   489k      0 --:--:-- --:--:-- --:--:--  489k
extracting libxml2-python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxml2-python-static@2.13.3 if necessary...
Found libxml2-python-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxml2-python-static/libxml2-python-static_2.13.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 72628  100 72628    0     0   449k      0 --:--:-- --:--:-- --:--:--  451k
extracting libxml2-python-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland@1.22.0-1 if necessary...
Found libwayland in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland/libwayland_1.22.0-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  115k  100  115k    0     0  1498k      0 --:--:-- --:--:-- --:--:-- 1514k
extracting libwayland to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libwayland-cross-scanner@1.22.0-1 if necessary...
Found libwayland-cross-scanner in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libw/libwayland-cross-scanner/libwayland-cross-scanner_1.22.0-1_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 81144  100 81144    0     0  1100k      0 --:--:-- --:--:-- --:--:-- 1116k
extracting libwayland-cross-scanner to /home/builder/.termux-build/_cache-all...
Downloading dependency libxrandr@1.5.4 if necessary...
Found libxrandr in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libx/libxrandr/libxrandr_1.5.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 21824  100 21824    0     0   303k      0 --:--:-- --:--:-- --:--:--  304k
extracting libxrandr to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency spirv-headers@1.3.290.0 if necessary...
Found spirv-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-headers/spirv-headers_1.3.290.0_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  129k  100  129k    0     0  1696k      0 --:--:-- --:--:-- --:--:-- 1701k
extracting spirv-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency spirv-tools@1.3.290.0 if necessary...
Found spirv-tools in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/spirv-tools/spirv-tools_1.3.290.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1793k  100 1793k    0     0  1794k      0 --:--:-- --:--:-- --:--:-- 1793k
extracting spirv-tools to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency vulkan-headers@1.3.293 if necessary...
Found vulkan-headers in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-headers/vulkan-headers_1.3.293_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 1312k  100 1312k    0     0  13.5M      0 --:--:-- --:--:-- --:--:-- 13.6M
extracting vulkan-headers to /home/builder/.termux-build/_cache-all...
Downloading dependency vulkan-utility-libraries@1.3.293 if necessary...
Found vulkan-utility-libraries in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/v/vulkan-utility-libraries/vulkan-utility-libraries_1.3.293_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  6  471k    6 30630    0     0  57400      0  0:00:08 --:--:--  0:00:08 57359
100  471k  100  471k    0     0   583k      0 --:--:-- --:--:-- --:--:--  583k
extracting vulkan-utility-libraries to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v1.3.293.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 2096k    0 2096k    0     0  4548k      0 --:--:-- --:--:-- --:--:-- 4548k
100 5183k    0 5183k    0     0  5484k      0 --:--:-- --:--:-- --:--:-- 6379k
Applying patch: 0001-no-android.patch
Applying patch: 0002-getenv.patch
Applying patch: 0003-layers-utils-android_ndk_types.h.patch
Downloading https://github.com/Kitware/CMake/releases/download/v3.29.3/cmake-3.29.3-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  4 50.7M    4 2271k    0     0  7970k      0  0:00:06 --:--:--  0:00:06 7970k
100 50.7M  100 50.7M    0     0  82.9M      0 --:--:-- --:--:-- --:--:--  148M
Downloading https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  120k  100  120k    0     0   209k      0 --:--:-- --:--:-- --:--:--  372k
Archive:  /home/builder/.termux-build/vulkan-validation-layers/tmp/ninja-1.12.1.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.12.1/ninja  
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The CXX compiler identification is Clang 18.0.1
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find robin_hood (missing: robin_hood_DIR)
-- Configuring done (0.4s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_C_FLAGS
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/vulkan-validation-layers/build
[1/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/error_location_helper.cpp.o
[2/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/error_location.cpp.o
[3/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/pnext_chain_extraction.cpp.o
[4/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/feature_requirements_helper.cpp.o
[5/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_object_types.cpp.o
[6/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/error_message/logging.cpp.o
[7/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/convert_utils.cpp.o
[8/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/hash_util.cpp.o
[9/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/image_layout_utils.cpp.o
[10/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_extension_utils.cpp.o
[11/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_extension_helper.cpp.o
[12/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/ray_tracing_utils.cpp.o
[13/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_dispatch_table_helper.cpp.o
[14/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_layer_utils.cpp.o
[15/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/utils/vk_struct_compare.cpp.o
[16/173] Building CXX object layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o
FAILED: layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27-api-24-v1/sysroot -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o -MF layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o.d -o layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/bindless_descriptor_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:266:19: error: no member named 'OpUntypedVariableKHR' in namespace 'spv'
  266 |         case spv::OpUntypedVariableKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:267:19: error: no member named 'OpUntypedAccessChainKHR' in namespace 'spv'
  267 |         case spv::OpUntypedAccessChainKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:268:19: error: no member named 'OpUntypedInBoundsAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsAccessChain'?
  268 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1482:5: note: 'OpInBoundsAccessChain' declared here
 1482 |     OpInBoundsAccessChain = 66,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:271:19: error: no member named 'OpUntypedPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  271 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:272:19: error: no member named 'OpUntypedInBoundsPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  272 |         case spv::OpUntypedInBoundsPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:273:19: error: no member named 'OpUntypedArrayLengthKHR' in namespace 'spv'
  273 |         case spv::OpUntypedArrayLengthKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:268:14: error: duplicate case value 'OpInBoundsAccessChain'
  268 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:63:14: note: previous case defined here
   63 |         case spv::OpInBoundsAccessChain:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:271:14: error: duplicate case value 'OpInBoundsPtrAccessChain'
  271 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:66:14: note: previous case defined here
   66 |         case spv::OpInBoundsPtrAccessChain:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:272:14: error: duplicate case value 'OpInBoundsPtrAccessChain'
  272 |         case spv::OpUntypedInBoundsPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:271:14: note: previous case defined here
  271 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:42:23: error: no return statement in constexpr function
   42 | static constexpr bool OpcodeHasType(uint32_t opcode) {
      |                       ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:646:19: error: no member named 'OpTypeUntypedPointerKHR' in namespace 'spv'
  646 |         case spv::OpTypeUntypedPointerKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:647:19: error: no member named 'OpUntypedVariableKHR' in namespace 'spv'
  647 |         case spv::OpUntypedVariableKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:648:19: error: no member named 'OpUntypedAccessChainKHR' in namespace 'spv'
  648 |         case spv::OpUntypedAccessChainKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:649:19: error: no member named 'OpUntypedInBoundsAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsAccessChain'?
  649 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1482:5: note: 'OpInBoundsAccessChain' declared here
 1482 |     OpInBoundsAccessChain = 66,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:652:19: error: no member named 'OpUntypedPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  652 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:653:19: error: no member named 'OpUntypedInBoundsPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  653 |         case spv::OpUntypedInBoundsPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/bindless_descriptor_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:654:19: error: no member named 'OpUntypedArrayLengthKHR' in namespace 'spv'
  654 |         case spv::OpUntypedArrayLengthKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:649:14: error: duplicate case value 'OpInBoundsAccessChain'
  649 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:441:14: note: previous case defined here
  441 |         case spv::OpInBoundsAccessChain:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:652:14: error: duplicate case value 'OpInBoundsPtrAccessChain'
  652 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:444:14: note: previous case defined here
  444 |         case spv::OpInBoundsPtrAccessChain:
      |              ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
[17/173] Building CXX object layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/buffer_device_address_pass.cpp.o
FAILED: layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/buffer_device_address_pass.cpp.o 
/home/builder/.termux-build/_cache/android-r27-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r27-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r27-api-24-v1/sysroot -DVK_ENABLE_BETA_EXTENSIONS -DVK_USE_PLATFORM_WAYLAND_KHR -DVK_USE_PLATFORM_XCB_KHR -DVK_USE_PLATFORM_XLIB_KHR -DVK_USE_PLATFORM_XLIB_XRANDR_EXT -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/. -I/home/builder/.termux-build/vulkan-validation-layers/src/layers -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan -I/home/builder/.termux-build/vulkan-validation-layers/src/layers/. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -Wpointer-arith -Wconversion -Wimplicit-fallthrough -Wstring-conversion -Wno-sign-conversion -Wno-implicit-int-conversion -MD -MT layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/buffer_device_address_pass.cpp.o -MF layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/buffer_device_address_pass.cpp.o.d -o layers/gpu/spirv/CMakeFiles/gpu_av_spirv.dir/buffer_device_address_pass.cpp.o -c /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:266:19: error: no member named 'OpUntypedVariableKHR' in namespace 'spv'
  266 |         case spv::OpUntypedVariableKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:267:19: error: no member named 'OpUntypedAccessChainKHR' in namespace 'spv'
  267 |         case spv::OpUntypedAccessChainKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:268:19: error: no member named 'OpUntypedInBoundsAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsAccessChain'?
  268 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1482:5: note: 'OpInBoundsAccessChain' declared here
 1482 |     OpInBoundsAccessChain = 66,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:271:19: error: no member named 'OpUntypedPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  271 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:272:19: error: no member named 'OpUntypedInBoundsPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  272 |         case spv::OpUntypedInBoundsPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:273:19: error: no member named 'OpUntypedArrayLengthKHR' in namespace 'spv'
  273 |         case spv::OpUntypedArrayLengthKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:268:14: error: duplicate case value 'OpInBoundsAccessChain'
  268 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:63:14: note: previous case defined here
   63 |         case spv::OpInBoundsAccessChain:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:271:14: error: duplicate case value 'OpInBoundsPtrAccessChain'
  271 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:66:14: note: previous case defined here
   66 |         case spv::OpInBoundsPtrAccessChain:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:272:14: error: duplicate case value 'OpInBoundsPtrAccessChain'
  272 |         case spv::OpUntypedInBoundsPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:271:14: note: previous case defined here
  271 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:42:23: error: no return statement in constexpr function
   42 | static constexpr bool OpcodeHasType(uint32_t opcode) {
      |                       ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:646:19: error: no member named 'OpTypeUntypedPointerKHR' in namespace 'spv'
  646 |         case spv::OpTypeUntypedPointerKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:647:19: error: no member named 'OpUntypedVariableKHR' in namespace 'spv'
  647 |         case spv::OpUntypedVariableKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:648:19: error: no member named 'OpUntypedAccessChainKHR' in namespace 'spv'
  648 |         case spv::OpUntypedAccessChainKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:649:19: error: no member named 'OpUntypedInBoundsAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsAccessChain'?
  649 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1482:5: note: 'OpInBoundsAccessChain' declared here
 1482 |     OpInBoundsAccessChain = 66,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:652:19: error: no member named 'OpUntypedPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  652 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:653:19: error: no member named 'OpUntypedInBoundsPtrAccessChainKHR' in namespace 'spv'; did you mean 'OpInBoundsPtrAccessChain'?
  653 |         case spv::OpUntypedInBoundsPtrAccessChainKHR:
      |              ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                   OpInBoundsPtrAccessChain
/data/data/com.termux/files/usr/include/spirv/unified1/spirv.hpp:1486:5: note: 'OpInBoundsPtrAccessChain' declared here
 1486 |     OpInBoundsPtrAccessChain = 70,
      |     ^
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/buffer_device_address_pass.cpp:17:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/module.h:21:
In file included from /home/builder/.termux-build/vulkan-validation-layers/src/layers/gpu/spirv/type_manager.h:19:
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:654:19: error: no member named 'OpUntypedArrayLengthKHR' in namespace 'spv'
  654 |         case spv::OpUntypedArrayLengthKHR:
      |              ~~~~~^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:649:14: error: duplicate case value 'OpInBoundsAccessChain'
  649 |         case spv::OpUntypedInBoundsAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:441:14: note: previous case defined here
  441 |         case spv::OpInBoundsAccessChain:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:652:14: error: duplicate case value 'OpInBoundsPtrAccessChain'
  652 |         case spv::OpUntypedPtrAccessChainKHR:
      |              ^
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vulkan/generated/spirv_grammar_helper.h:444:14: note: previous case defined here
  444 |         case spv::OpInBoundsPtrAccessChain:
      |              ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
[18/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vk_layer_config.cpp.o
/home/builder/.termux-build/vulkan-validation-layers/src/layers/vk_layer_config.cpp:72:13: warning: unused function 'PropCallback' [-Wunused-function]
   72 | static void PropCallback(void *cookie, [[maybe_unused]] const char *name, const char *value, [[maybe_unused]] uint32_t serial) {
      |             ^~~~~~~~~~~~
1 warning generated.
[19/173] Building CXX object layers/CMakeFiles/VkLayer_utils.dir/vulkan/generated/vk_function_pointers.cpp.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-08-21 00:44:41 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/10481182906">10481182906</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/vulkan-validation-layers/build.sh
+++ b/packages/vulkan-validation-layers/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/KhronosGroup/Vulkan-ValidationLayers
 TERMUX_PKG_DESCRIPTION="Vulkan Validation Layers"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.3.290"
+TERMUX_PKG_VERSION="1.3.297"
 TERMUX_PKG_SRCURL=https://github.com/KhronosGroup/Vulkan-ValidationLayers/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=37fd611f1c5c5879a68c6f63b09e600a3858f519aa147a2535961fa3d8e889ae
+TERMUX_PKG_SHA256=11220ccc139f5517b3a3aa2e2ef0d680da314940c9bb98c7f3857a871bdb2205
 TERMUX_PKG_DEPENDS="libc++, vulkan-loader"
 TERMUX_PKG_BUILD_DEPENDS="libwayland, libx11, libxcb, libxrandr, spirv-headers, spirv-tools, vulkan-headers (=${TERMUX_PKG_VERSION}), vulkan-utility-libraries (=${TERMUX_PKG_VERSION})"
 TERMUX_PKG_ANTI_BUILD_DEPENDS="vulkan-loader"

